pkg/framework/plugins/removepodshavingtoomanyrestarts/toomanyrestarts_test.go

```diff
@@ -121,100 +120,115 @@ func TestRemovePodsHavingTooManyRestarts(t *testing.T) {
 	tests := []struct {
 		description                    string
 		nodes                          []*v1.Node
-		strategy                       api.DeschedulerStrategy
+		args                           componentconfig.RemovePodsHavingTooManyRestartsArgs
 		expectedEvictedPodCount        uint
 		maxPodsToEvictPerNode          *uint
 		maxNoOfPodsToEvictPerNamespace *uint
+		nodeFit                        bool
 	}{
 		{
 			description:             "All pods have total restarts under threshold, no pod evictions",
-			strategy:                createStrategy(true, true, 10000, false),
+			args:                    createRemovePodsHavingTooManyRestartsAgrs(10000, true),
 			nodes:                   []*v1.Node{node1},
 			expectedEvictedPodCount: 0,
+			nodeFit:                 false,
```

Comment:
This line can be omitted, `false` is the default value for `nodeFit`. The same for other lines.